packages/tabs/src/Default/Default.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ThemeProvider } from 'react-bootstrap';
+import Tab from 'react-bootstrap/Tab';
+import Tabs from 'react-bootstrap/Tabs';
+import { TabsDefaultProps } from './Default.types';
+import * as styleMod from './styles.module.scss';
+import { createLocalProps } from '@owlui/utils';
+
+export const Component = (props: TabsDefaultProps) => {
+  const baseClass = 'owluiTabs';
+  const { children, defaultActiveKey, items } = props;
+  const prefix = props.prefix || '';
+
+  const locals = createLocalProps(
+    props,
+    {
+      module: styleMod,
+      classes: {
+        base: baseClass,
+        prefix: prefix,
+        optionals: [
+          {
+            fields: ['theme'],
+            value: 'Theme',
+          },
+          {
+            fields: ['theme', 'appearance'],
+            value: 'Theme',
+          },
+          {
+            fields: ['size'],
+            value: 'Size',
+          },
+        ],
+      },
+    },
+    ['prefix', 'theme', 'appearance', 'size', 'tabs']
```

Comment:
We also want to add `items` to this list. All props passed to the component are renderer as properties in the HTML element, this list here contains the `removables`, things we pass as props and we don't want to render, such as the `items`:

![image](https://user-images.githubusercontent.com/98419841/172704295-07a07236-8cde-45a7-aa0b-c2d986ec7aad.png)